Makefile

```diff
@@ -86,13 +45,19 @@ deploy:
 	$(DOCKER_COMPOSE) pull $(service)
 	$(DOCKER_COMPOSE) stop $(service)
 	$(DOCKER_COMPOSE) up -d $(service)
+ifdef POST_DEPLOY_SCRIPT
+	chmod +x ${CAMINO_HOME}/conf/camino/${POST_DEPLOY_SCRIPT} && ${CAMINO_HOME}/conf/camino/${POST_DEPLOY_SCRIPT}
+endif
 	$(DOCKER_COMPOSE) restart nginx
 
 .PHONY: deploy-all-service
 deploy-all-service:
```

Comment:
We can rename `deploy-all-service` to `deploy-all` to maintain parity with existing deploy cycles